pkgs/development/python-modules/rbtools/default.nix

```diff
@@ -16,10 +16,10 @@ buildPythonPackage rec {
     sha256 = "1ng8l8cx81cz23ls7fq9wz4ijs0zbbaqh4kj0mj6plzcqcf8na4i";
   };
 
-  buildInputs = [ nose ];
+  checkInputs = [ nose ];
   propagatedBuildInputs = [ six ];
 
-  checkPhase = "nosetests";
+  checkPhase = "LC_ALL=C nosetests";
```

Comment:
Why was setting `LC_ALL` needed? 
We have https://github.com/NixOS/nixpkgs/commit/1fccd255953b6025a4d0342c2cf5641e5b6b6ced

Reply:
A test failed without it:

```
running install tests
..........E..........................SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.................SSSSSSSSSSSSSSSSSSS...............
======================================================================
ERROR: Testing the cache with the Expires header in the future
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/RBTools-0.7.2/rbtools/api/tests.py", line 782, in test_expires_header_future
    locale.resetlocale(locale.LC_TIME)
  File "/nix/store/ih03q5vi6kj278l4jk4kh41lqyagzzdi-python-2.7.15/lib/python2.7/locale.py", line 591, in resetlocale
    _setlocale(category, _build_localename(getdefaultlocale()))
Error: unsupported locale setting
```

When I run it with "LC_ALL=C.UTF-8 nosetests" it fails, with LC_ALL=C it doesn't. I have no idea why.